homeassistant/components/suez_water/sensor.py

```diff
@@ -23,39 +27,37 @@
 })
 
 
-def setup_platform(hass, config, add_devices, discovery_info=None):
+def setup_platform(hass, config, add_entities, discovery_info=None):
     """Set up the sensor platform."""
-    from pysuez.client import PySuezError
-    from pysuez import SuezClient
-
     username = config[CONF_USERNAME]
     password = config[CONF_PASSWORD]
     counter_id = config[CONF_COUNTER_ID]
     try:
         client = SuezClient(
             username, password, counter_id)
 
-        if client.check_credentials():
-            add_devices([SuezSensor(client)], True)
-        else:
+        if not client.check_credentials():
             _LOGGER.warning("Wrong username and/or password")
+        else:
```

Comment:
`return` if the credentials are bad.

Reply:
`pylint` does not like a `return` followed by `else`. Should I leave it anyway?

Reply:
Remove the else and outdent after the return.

Reply:
Done.